src/wiseagents/core.py

```diff
@@ -1107,14 +1123,14 @@ def does_context_exist(cls, context_name: str) -> bool:
                 return True
     
     @classmethod
-    def remove_agent(cls, agent_name: str):
+    def unregister_agent(cls, agent_name: str):
         """
-        Remove the agent from the registry
+        Remove the agent from the registry this should be used only on agents which already stopped STOMP connection
```

Comment:
Very minor but instead of referencing STOMP explicitly here probably better to be more general and just refer to transport instead.